src/components/CaseEditor.js

```diff
@@ -3,7 +3,9 @@ import React, {Component} from 'react'
 import {injectIntl, intlShape} from 'react-intl'
 import { Field } from 'redux-form'
 
+import '../containers/Case/Case.css'
```

Comment:
This seems wrong.  I suspect the CSS needs to be in `components`, not in `containers`.